README.md

```diff
@@ -33,7 +47,7 @@ My Solution
 I write C code. I use linux, it seemed only reasonable that as a version
 didn’t exist in this space I would create it and open-source it for all
 to use (and improve) You can get the sources from [GitHub] and build it
-yourself by following the instructions at my [xhk page][xHK] Once the
+yourself by following the instructions at my [xhk page][xHK]. Once the
```

Comment:
We should change this to stop referring to my blog ... although it's still active at www.kieranbingham.co.uk... it's a missing redirect service it would seem.

Lets split the documentation change and code changes into two patches for clarity as well.